content/docs/user-guide/gems/reference/aws/aws-gamelift/advanced-topics.md

```diff
@@ -134,6 +134,6 @@ C:\game\bin\server.exe --engine-path=C:\game --project-path=C:\game\process1 --p
 
 ---
 
-Previous topic: [AWS GameLift Gem C++ API](cpp-api/)
+Previous topic: [FlexMatch Scripting](flexmatch/scripting/)
```

Comment:
I'm not sure why this section has manually defined "Previous topic" and "Next topic" links, but I think we should remove all these in favor of some sort of automation in the page design. Might be worth a GHI or two if you feel like making them (e.g. one to remove the manual links and another to design built-in "previous/next" links on the page).

Reply:
Yes, I was debating what to do about next/previous links as well. For this section (GameLift), I recall incoming PRs specifically defining these links, and as they were consistent across other pages in this section, I decided to keep them in. 

I do lean towards voting to remove the links, as it's not something we have in other sections. Nor do I think it helps with navigation for this section, as the pages can be viewed in different orders. And as shown in this PR, they can easily become outdated. I can create an issue for that. 

As for adding built-in previous/next navigation links, I can create an issue for that to begin and track a discussion regarding whether or not we want that feature.